libs/localization/src/lib/en.json

```diff
@@ -1264,8 +1264,8 @@
       "shiftCohortDescription": "Select a cohort from the cohort list. Apply the cohort to the dashboard."
     },
     "CohortInformation": {
-      "SwitchGlobalCohort": "Switch global cohort",
-      "CreateNewCohort": "Create new cohort",
+      "SwitchGlobalCohort": "Shift cohort",
+      "CreateNewCohort": "Save cohort",
```

Comment:
should the variable names be updated too:
CreateNewCohort -> SaveCohort
SwitchGlobalCohort -> ShiftCohort